db/migrate/20170418064453_add_timestamp_to_custom_fields.rb

```diff
@@ -1,9 +1,9 @@
 class AddTimestampToCustomFields < ActiveRecord::Migration[5.0]
   def change
-    add_column :custom_fields, :created_at, :datetime
-    add_column :custom_fields, :updated_at, :datetime
+    add_column :custom_fields, :created_at, :datetime unless column_exists? :custom_fields, :created_at
```

Comment:
Good point (albeit it being uttered in a non lingua franca). Will add a specific down method.